app/views/submissions/viewPDF.html.erb

```diff
@@ -31,9 +31,9 @@
     var annotations = <%=raw @annotations.to_json %>;
     var fileNameStr = "<%= @filename %>";
     <% if @preview_mode then %>
-    var fileLink    = "<%= download_course_assessment_submission_path(@course, @assessment, @submission, :annotated => true) %>";
+    var fileLink    = "<%= download_course_assessment_submission_path(@course, @assessment, @submission, :header_position =>params[:header_position], :annotated => true) %>";
     <% else %>
-    var fileLink    = "<%= download_course_assessment_submission_path(@course, @assessment, @submission) %>";
+    var fileLink    = "<%= download_course_assessment_submission_path(@course, @assessment, @submission, :header_position =>params[:header_position]) %>";
```

Comment:
I would use the new symbol syntax here.  Also, does this fix the downloading submission errors?  If not, I would hide the download link until that is fixed.